packages/devextreme/js/__internal/ui/chat/messagebubble.ts

```diff
@@ -47,8 +47,9 @@ class MessageBubble extends Widget<Properties> {
       isDeleted = false,
       template,
     } = this.option();
-    const $bubbleContainer = $(this.element()).find(`.${CHAT_MESSAGEBUBBLE_CONTENT_CLASS}`);
+    this.$element().removeClass(CHAT_MESSAGEBUBBLE_DELETED_CLASS);
```

Comment:
There is no tests for checking that class add/remove after change Message.isDeleted option  at runtime